flaml/nlp/utils.py

```diff
@@ -85,13 +89,19 @@ def tokenize_and_align_labels(
     Y_sent_key=None,
     return_column_name=False,
 ):
+    if hf_args.max_seq_length is not None:
+        logger.warning(
```

Comment:
shall this warning be issued only if task is token classification?

Reply:
I think it's within the tokenize_and_align_labels function so only for token classification task.